docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,15 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+Ledger
+~~~~~~
+
+* Do not compile packages twice when loading a scenario with ``--scenario``, see
```

Comment:
This isn't super clear from a user point of view. I'm guessing it should be something like "When loading a scenario, the sandbox no longer compiles packages twice"?

Reply:
Changed to match your wording